app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -47,14 +50,28 @@
                         android:layout_weight="70"
                         android:orientation="vertical">
 
-                        <TextView
-                            android:layout_width="match_parent"
+                        <LinearLayout
```

Comment:
This layout should have constraints since it is in a ConstraintLayout.

Reply:
The top level NestedScrollView has the constraints in this file, but I'll update the new layouts where they are missing